### PR TITLE
fixed plaintext function

### PR DESCRIPTION
**Pull request by debonair**
Fixed issue with daily and weekly reports

### CODE REVIEW

**Comment by debonair** [comment]
@rcrichton please review the plaintext method was returning an object and not plain text so it was breaking the email send.

File: src/reports.coffee [hunk 1]
```diff
@@ -197,10 +200,8 @@ setupAgenda = (agenda) ->
   agenda.every "1 days", 'send daily channel metrics'
 
 exports.setupAgenda = setupAgenda
-
-if process.env.NODE_ENV == "test"
```
Comment (rcrichton) [inline src/reports.coffee]: Could you add this back in. I don't think relates to the chance that was made.

**Comment by rcrichton** [comment]
Hey @debonair looks good to me. Just add back the check for the test env as mentioned above, then :+1:

File: src/reports.coffee (hunk 1 above)
Comment (debonair) [inline src/reports.coffee]: Ok cool

**Comment by debonair** [comment]
closes #466

**Pull request merged**